db/models.py

```diff
@@ -1 +1,53 @@
 from django.db import models
+from django.db.models import CharField
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True,)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE,
+        related_name="skill_set"
```

Comment:
There is no need to define `related_name` value equal to a default value, use `skills` as value instead